.gitignore

```diff
@@ -6,3 +6,4 @@ vendor/*
 aiven/aiven-packr.go
 packrd/*
 plugins/
+.idea/
```

Comment:
I would recommend adding developer's ide (or enviroment specific) ignores to your global .gitignore

Reply:
good idea, thanks